src/conv.rs

```diff
@@ -207,7 +207,17 @@ pub fn map_device_descriptor<'a>(
                 label: OwnedLabel::new(extras.label).into_cow(),
                 features: features,
                 limits: wgt::Limits {
-                    max_bind_groups: extras.maxBindGroups,
+                    max_texture_dimension_1d: extras.maxTextureDimension1D | 8192,
```

Comment:
instead of hard-coding the base limits, we should take `wgt::Limits::default()` and use its values

Reply:
Good point.